### ISSUE TITLE
Don't run as root

### ISSUE THREAD

**Issue by consideRatio** [jupyterhub/docker-image-cleaner#8]
Our Dockerfile runs the script as root, we shouldn't.

**Comment by yuvipanda** [comment]
Nope, we can run as non-root as long as the docker socket is accessible. Access to the docker socket does make us 'root equivalent' though. We might have to belong to the 'docker' group maybe? I don't know what permissions bind mounted docker socket usually has

**Comment by consideRatio** [comment]
This can probably then be closed by documenting in #7, as we may require being root.

> . Access to the docker socket does make us 'root equivalent' though.

Does it matter if the docker socket is exposed via another Pod, a daemonset on the node, rather than being directly installed on the node? Or is it? Hmm... I'm very vague about these matters and would love to acquire some overview about this.

**Comment by minrk** [comment]
It needs access to the docker socket _and_ access to measure disk usage (~`du -hs /var/lib/docker`). This usually requires root.
